### DISCUSSION TITLE
App for focus time

### DISCUSSION THREAD

**Discussion by tenisaname** [TomSchimansky/CustomTkinter#1241] [Showcase]
Hi guys, I created an app for focus time
The focus time is 1 hour at the end of the hour a pop up pops out
Tell me what you would like to see in such an application?
![image](https://user-images.githubusercontent.com/60942207/219410468-57d24f79-0efe-4bdb-bd48-3d058113ab88.png)
https://github.com/tenisaname/forfocustime

**Comment by saiyam-sandhir** [comment]
> Hi guys, I created an app for focus time The focus time is 1 hour at the end of the hour a pop up pops out Tell me what you would like to see in such an application? ![image](https://user-images.githubusercontent.com/60942207/219410468-57d24f79-0efe-4bdb-bd48-3d058113ab88.png) https://github.com/tenisaname/forfocustime

You can try making a Pomodoro app
![62e61034-b66a-4ede-a5db-a5cbc39d55e7_SP+587+-+The+Pomodoro+technique](https://user-images.githubusercontent.com/86138251/219934443-fb755313-4735-4de9-9629-69f641339d8b.png)

**Reply by saiyam-sandhir** [reply]
long break is usually 15 min
